content/en/guides/rhinocommon/installing-tools-mac/index.md

```diff
@@ -37,47 +37,26 @@ This guide presumes you have an:
-1. If you do not have Xcode installed, the installer may prompt you to install it. You can ignore this warning.
-1. The installer downloads and installs *Visual Studio for Mac* and any additional components you checked.
-1. When the installer is finished, click the *Launch Visual Studio* button.
-1. Visual Studio is installed in your */Applications* folder. You will want to *drag its icon to your Dock* for future use or - if it's running - right/option-click the icon in the Dock and select *Keep in Dock*.
-
-## Install the Rhino Visual Studio Extension
-
-The Rhino AddIn/Extension is required to debug your plugin code in an active session of Rhino for Mac using Visual Studio for Mac. Additionally, it contains project templates to get you started creating plugins quickly.
-
-#### Step-by-Step
-
-1. Visit the [Extension's GitHub releases page](https://github.com/mcneel/RhinoVisualStudioExtensions/releases) and find the *Latest release* in the [list of releases](https://github.com/mcneel/RhinoVisualStudioExtensions/releases).
-1. Download the *.mpack* file in the list of Downloads on that release. For example, at the time of this writing, the *Latest release* download is entitled *Rhino.VisualStudio.Mac.2022-8.0.0.mpack*. (If you are using Visual Studio for Mac 2019, you will want to get *Rhino.VisualStudio.Mac.2019-7.21.0.mpack*).
-1. Launch *Visual Studio for Mac* if it not already open.
-1. Navigate to *Visual Studio* > *Extensions...*...
-1. Click the *Install from file...* button in the lower right-hand corner.
-1. Navigate to the *.mpack* file you downloaded in step 2 above.
-1. Click *Install*. The plugin should install.
-1. *IMPORTANT*: You must *Quit* and *Restart* Visual Studio for Mac.
-1. Navigate to *Visaul Studio* > *Extensions...* > *Installed* tab. Verify that *RhinoCommon Plugin Support* exists. If it's there, you have successfully installed the Extension and you are *DONE*.
+1. *[Download Visual Studio Code](https://code.visualstudio.com/)*.
+1. Once you have downloaded the *VSCode-darwin-universal.zip*, double-click it to unzip.
+1. Drag Visual Studio Code to the */Applications* folder.
+1. Open Visual Studio Code, you will need to click "Open" the first time you open it.
+1. You will need to install a number of packages before starting. These can be found by clicking "Extensions" on the left side bar:
+   - [C#](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+   - [C# Dev kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit)
+   - [Intellicode for C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscodeintellicode-csharp)
+   - [.NET Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-pack)
+   - [.NET Install Tool](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime&ssr=false#overview) (*Recommended*)
```

Comment:
Do we need to mention .NET Install Tool?  It is used by extensions to ensure .NET is installed and not really user-facing.

The only real required extension is C# Dev Kit, which includes C# and .NET Install Tool.

Reply:
the .NET install tool is included by the C# dev kit.
Intellicode is definitely a great install suggestion

I wanted to list everything that would get installed. I could Just list the Intellicode for C# Dev Kit which would install everything necessary, I think the .NET Extension Pack is unnecessary though